x-pack/test/functional/services/ml/job_table.ts

```diff
@@ -373,6 +373,19 @@ export function MachineLearningJobTableProvider(
       await testSubjects.existOrFail('~mlPageJobWizard');
     }
 
+    public async clickCloneJobActionWhenNoDataViewExists(jobId: string) {
+      await this.ensureJobActionsMenuOpen(jobId);
+      await testSubjects.click('mlActionButtonCloneJob');
+      await this.assertNoDataViewForCloneJobWarningToastExist();
+    }
+
+    public async assertNoDataViewForCloneJobWarningToastExist() {
+      await retry.tryForTime(5000, async () => {
+        const toast = await testSubjects.find('mlCloneJobNoDataViewExistsWarningToast');
+        expect(toast).not.to.be(undefined);
```

Comment:
do we need to check the content of the toast?

Reply:
I think it's ok just to check the `data-test-subj` for now, and this is in line with other tests we have for toast notifications.